### ISSUE TITLE
It should be possible to install from the full commit SHA

### ISSUE THREAD

**Issue by ematipico** [stackblitz-labs/pkg.pr.new#93]
At the moment, pkg.pr.new publishes packages using the short version of the commit SHA, however it should be possible to retrieve it/install it using the full commit SHA:

```
@biomejs/wasm-web: npm i https://pkg.pr.new/biomejs/website/@biomejs/wasm-web@d7a4398
```

```
Run pnpm i https://pkg.pr.new/biomejs/biome/@biomejs/wasm-web@d7a4398868fd71842cc7db9cbf99aa01db07849b
Progress: resolved 0, reused 0, downloaded 1, added 0
 ERR_PNPM_FETCH_404  GET https://pkg.pr.new/biomejs/biome/@biomejs/wasm-web@d7a4398868fd71842cc7db9cbf99aa01db07849b: Not Found - 404
```

**Comment by ematipico** [comment]
For more context, during the GitHub action workflow, I am not able to retrieve the short version of `github.sha` unless I use some other action that does the job. Maybe there's a global variable that I am not aware of?

**Comment by Aslemammad** [comment]
Hey, yea makes sense, but I wonder if you can abbreviate the commit hash (sha.substring(0, 7)) and use that. would it solve your issue for now?

**Comment by ematipico** [comment]
Yeah, that's a workaround, but I am not very good with CI scripts, or bash scripts, so I wouldn't know how to do that without losing another 1h 😓

**Comment by Aslemammad** [comment]
Ok I'll find a way through this and I'll let you know!

**Comment by Aslemammad** [comment]
just fixed this, will release soon!

**Comment by ematipico** [comment]
That was fast! Thank you

**Comment by Aslemammad** [comment]
🙌🙌🙌

**Comment by Aslemammad** [comment]
just released 0.0.8!
